scripts/delayed.js

```diff
@@ -4,5 +4,22 @@ import { loadScript, sampleRUM } from './aem.js';
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
+function getEnvType(hostname = window.location.hostname) {
```

Comment:
`icicidirect.com` and `www.icicidirect.com` should also be considered. Probably we can map them to `prod`